WinUIGallery/Controls/DesignGuidance/ColorSections/SignalSection.xaml

```diff
@@ -0,0 +1,170 @@
+<!--  Copyright (c) Microsoft Corporation and Contributors.  -->
+<!--  Licensed under the MIT License.  -->
+
+<Page
+    x:Class="WinUIGallery.DesktopWap.Controls.DesignGuidance.ColorSections.SignalSection"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:designguidance="using:WinUIGallery.DesktopWap.Controls.DesignGuidance"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d">
+
+
+    <StackPanel>
+        <!--  System  -->
+        <designguidance:ColorPageExample
+            Title="System"
+            Background="{ThemeResource SolidBackgroundFillColorQuarternaryBrush}"
+            Description="Used for accent fills on controls">
+            <InfoBar
+                Title="Title"
+                IsOpen="True"
+                Message="This is body text. Windows 11 is faster and more intuitive."
+                Severity="Error" />
+        </designguidance:ColorPageExample>
+
+        <Grid Style="{StaticResource ColorTilesPanelMiddleStyle}">
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition Width="*" />
+                <ColumnDefinition Width="*" />
+                <ColumnDefinition Width="*" />
+            </Grid.ColumnDefinitions>
+            <designguidance:ColorTile
+                Background="{ThemeResource SystemFillColorAttentionBrush}"
+                ColorBrushName="SystemFillColorAttentionBrush"
+                ColorExplanation="Badge"
+                ColorName="System / Attention"
+                ColorValue="#60CDFF (100%)"
+                Foreground="{ThemeResource TextFillColorInverseBrush}"
+                ShowSeparator="False" />
```

Comment:
Is there a specific reason to stick with the order provided in the Figma File compared to arranging them in a way that the Success, Caution and Critical files are next to each other and their respective background colors beneath them? I originally did that in #1167 since it looks more pleasing in my opinion.

Reply:
![image](https://user-images.githubusercontent.com/9866362/217235078-5ca39ee6-152d-43f1-b9a7-a716da39720f.png)

Yeah, there is a connection between those colors (one is the icon and other is the background of the same style so it makes sense to keep them together?). Moved the attention below similar to what you had so the green/orange/red is visually more pleasing :)